web-app/CKS.Web.Test/IntegrationTests/Infrastructure/CKSWebApplicationFactory.cs

```diff
@@ -25,6 +25,8 @@ public CKSWebApplicationFactory(ITestOutputHelper output)
 		{
 			_output = output;
 			_output.WriteLine("CKSWebApplicationFactory.constructor");
+
+			this.ClientOptions.BaseAddress = new Uri("https://cks-integration-tests.nice.org.uk");
```

Comment:
Just curious where is this site hosted?

Reply:
It's not. This is just to make the fake, in memory test server _think_ it's running a proper subdomain under nice.org.uk. Without this, .NET core doesn't let you set cookies with an arbitrary domain - it needs to match the request domain